bin/run-ci.sh

```diff
@@ -14,4 +14,5 @@ sbt -Dfile.encoding=UTF-8 \
   crossTestBridges \
   "publishBridges" \
   "$PROJECT_ROOT/test" \
-  "zincScripted/test:run"
+  "zincScripted/test:run" \
+  "zincScripted/test:run --to-jar"
```

Comment:
Could you add a new CI stage that runs this independently? We don't want to slow down CI by running the integration tests twice in the same Drone machine, it's better if we do it in parallel.

Reply:
Done